modules/document_repository/php/document_repository.class.inc

```diff
@@ -79,9 +79,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Date_uploaded',
                          );
 
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = $_GET['openAccordion'] ?? '';
         $this->tpl_data['openAccordion'] = $openAccordion;
-        $filtered = $_GET['filtered'];
+        $filtered = !empty($_GET['filtered']) ? true : false;
```

Comment:
`$filtered = !empty($_GET['filtered']);` is correct. Don't need the ternary operator now.